content/infra_language.md

```diff
@@ -1,48 +1,37 @@
 +++
-title = "About the Recipe DSL"
+title = "About the Chef Infra Language"
 draft = false
 
 gh_repo = "chef-web-docs"
 
-aliases = ["/dsl_recipe.html"]
+aliases = ["/dsl_recipe.html", "/dsl_recipe"]
 
 [menu]
   [menu.infra]
-    title = "DSL Overview"
-    identifier = "chef_infra/cookbook_reference/recipe_dsl/dsl_recipe.md DSL Overview"
-    parent = "chef_infra/cookbook_reference/recipe_dsl"
+    title = "Language Overview"
+    identifier = "chef_infra/cookbook_reference/infra_language/infra_language.md Language Overview"
+    parent = "chef_infra/cookbook_reference/infra_language"
     weight = 10
 +++
 
-{{% dsl_recipe_summary %}}
+{{% infra_lang_summary %}}
 
-Because the Recipe DSL is a Ruby DSL, anything that can be done using
-Ruby can also be done in a recipe or custom resource, including `if` and
-`case` statements, using the `include?` Ruby method, including recipes
-in recipes, and checking for dependencies. See the [Ruby
-Guide](/ruby/) for further information on built-in Ruby
-functionality.
+The Chef Infra Language is based on Ruby, allowing you to utilize the power of Ruby when the built-in language doesn't meet your needs out of the box. If you'd like to learn more about extending your Chef Infra code by using Ruby see our [Ruby Guide](/ruby/) for further information on Ruby functionality.
 
-## Include Recipes
+## Resources
 
-{{% cookbooks_recipe_include_in_recipe %}}
-
-### Reload Attributes
+Resources are the cornerstone of the Chef Infra Language. Resources define the desired state of an object on a system. A resource can be as simple as a directory or as complex or a complete security policy. Chef Infra Client ships with over 150 resources for configuring components such as packages, file, directories, or firewalls. For more information on resources in Chef Infra Client including a complete list of those included out of the box see [Resources](/resources).
```

Comment:
Can you give me something more specific term (or set of terms) than "object" in the "desired state of an object on a system" 

```suggestion
The Chef Infra Language expresses the desired state of configurable objects in your system in resources. They are the building blocks of recipes and they are central to understanding configuration management with Chef Infra. Some resources are as simple as a directory name, while others as complex as a complete security policy. Chef Infra Client ships with over 150 pre-packaged resources for configuring components such as packages, files, directories, and firewalls. Learn more in the Chef Infra Client [Resources](/resources) documentation.
```